frontend/tsconfig.json

```diff
@@ -0,0 +1,38 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "outDir": "build",
+    "module": "esnext",
```

Comment:
Used to be `commonjs`. Sure of that change?

Reply:
back to commonjs